README.md

```diff
@@ -159,26 +159,37 @@ An individual SDK will consider a feature implemented once the following require
 * For the forseeable future, each SDK is free to publish releases at a frequency that the SDK's respective DRI sees fit
 
 ## CI / CD
-Each SDK will use Github Actions for CI/CD
+Each SDK will use Github Actions for CI/CD and other automations
 
 | Feature                       | Typescript | Kotlin | Rust | Swift |
 | ----------------------------- | ---------- | ------ | ---- | ----- |
 | OSS License Check             | ❌          | ❌      | ❌    | ❌     |
+| Security Scanning             | ❌          | ❌      | ❌    | ❌     |
+| Static Analysis Linting/Style | ❌          | ❌      | ❌    | ❌     |
 | Running Unit Tests            | ✅          | ✅      | ❌    | ❌     |
+| Publishing Tests Reports      | ❌          | ❌      | ❌    | ❌     |
+| Code Coverage (CodeCov)       | ❌          | ❌      | ❌    | ❌     |
 | Publishing Artifacts          | ❓          | ❌      | ❌    | ❌     |
+| Release Template Checklist    | ❌          | ❌      | ❌    | ❌     |
+| Automated GH Release Tag      | ❌          | ❌      | ❌    | ❌     |
 | Publishing API Reference Docs | ❌          | ✅      | ❌    | ❌     |
 | Publish Example Feature Usage | ❌          | ✅      | ❌    | ❌     |
 
+* GitHub Actions should run in secured runners
```

Comment:
what is a "secured runner"?

Reply:
From the TBD Supply Chain Security doc:

> ** Authoritative Build Environment **
> A secure, authoritative build environment ensures software is compiled and packaged in a controlled, tamper-resistant setting. This mitigates the risk of introducing vulnerabilities or malicious code during the build process, whether through external attacks or compromised internal components. As a result, users and developers can trust that the distributed binaries and packages originate from a verified source and maintain the software's intended integrity.
>This can use similar tech (such as github actions) to build but ideally would be an isolated/gapped environment (many mainstream large OSS projects do this).

So they would be [secured self hosted ones](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners/about-self-hosted-runners).